content/handbook/support/workflows/community-support.md

```
@@ -0,0 +1,141 @@
+
+フォーラムでのコミュニティ支援への docs-first アプローチは、関連する回答／解決策を発見しやすく、すぐに利用できるようにすることで、これらの「特殊なタイプ」のギャップを埋めるのに役立ちます。
+
+たとえば、これらのコミュニティフォーラムスレッドでは、回答や解決策がドキュメントには「合わない」「特殊なタイプ」となっています:
+
+- 状況固有のチュートリアル／ハウツー -（例: 「DockerコンテナとしてGitLab CE 11.0を実行していて、ダウンタイムを最小限に抑えてGitLab 13.2 EE Omnibusにアップグレードしたいのですが、どうすれば安全にこれを行えますか?」）
+- コンテキストベースの説明 -（例: 「ドキュメントを読みましたが、Xを行う方法がわかりません。」）
+- 問題固有のトラブルシューティング手順（例: 「GitLab を 12.9 から 13.1 にアップグレードしたら動作しなくなり、すべてのページで 404 エラーが発生します - 助けてください！」）
+
+### コミュニティ支援ワークフロー
+
+- **[First Responder（初動対応者）](#first-responder)** - お客様向けの問題、バグ、リグレッションの早期発見
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)